pkg/sink/interceptors.go

```diff
@@ -0,0 +1,87 @@
+/*
+ Copyright 2019 The Tekton Authors
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package sink
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"golang.org/x/xerrors"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Retrieve ObjectReference to URI
+func GetURI(objRef *corev1.ObjectReference, ns string) (*url.URL, error) {
+	// TODO: This should work for any Adressable.
+	// Use something like https://github.com/knative/eventing-contrib/blob/7c0fc5cfa8bd44da0767d9e7b250264ea6eb7d8d/pkg/controller/sinks/sinks.go#L32
+	if objRef.Kind == "Service" && objRef.APIVersion == "v1" {
+		// TODO: Also assuming port 80 and http here. Use DNS/or the env vars?
```

Comment:
Can we add an additional line to the documentation that the interceptor service should specify port 80?